data-products/src/main/scala/org/sunbird/analytics/sourcing/SourcingSummaryReport.scala

```diff
@@ -0,0 +1,124 @@
+package org.sunbird.analytics.sourcing
+
+import java.util.Properties
+
+import org.apache.spark.SparkContext
+import org.sunbird.analytics.util.CourseUtils
+import org.apache.spark.sql.functions.{when, _}
+import org.apache.spark.sql._
+import org.ekstep.analytics.framework.fetcher.DruidDataFetcher
+import org.ekstep.analytics.framework.util.DatasetUtil.extensions
+import org.ekstep.analytics.framework.{DruidQueryModel, FrameworkContext, IJob, JobConfig, Level, StorageConfig}
+import org.ekstep.analytics.framework.util.{CommonUtil, JSONUtils, JobLogger, RestUtil}
+import org.sunbird.analytics.exhaust.BaseReportsJob
+import org.sunbird.cloud.storage.conf.AppConf
+
+case class SourcingContents(primaryCategory: String, createdBy: String, count: Int)
+
+object SourcingSummaryReport extends optional.Application with IJob with BaseReportsJob {
+
+  implicit val className = "org.sunbird.analytics.sourcing.SourcingSummaryReport"
+  val jobName: String = "SourcingSummaryReport"
+  val db = AppConf.getConfig("postgres.db")
+  val url = AppConf.getConfig("postgres.url") + s"$db"
+  val connProperties = CommonUtil.getPostgresConnectionProps
+
+  // $COVERAGE-OFF$ Disabling scoverage for main method
+  def main(config: String)(implicit sc: Option[SparkContext], fc: Option[FrameworkContext]): Unit = {
+    JobLogger.log(s"Started execution - $jobName", None, Level.INFO)
+    implicit val jobConfig = JSONUtils.deserialize[JobConfig](config)
+    implicit val frameworkContext: FrameworkContext = getReportingFrameworkContext()
+    implicit val spark = openSparkSession(jobConfig)
+
+    try {
+      val res = CommonUtil.time(execute())
+      JobLogger.end(s"$jobName completed execution", "SUCCESS", Option(Map("timeTaken" -> res._1)))
+    } finally {
+      frameworkContext.closeContext()
+      spark.close()
+    }
+  }
+
+  // $COVERAGE-ON$ Enabling scoverage for all other functions
+  def execute()(implicit spark: SparkSession, fc: FrameworkContext, config: JobConfig) = {
+    val tables = JSONUtils.deserialize[Map[String, String]](JSONUtils.serialize(config.modelParams.get.getOrElse("tables", Map())))
+    val programData = loadData(url, tables("programTable")).withColumnRenamed("status", "programStatus")
+    val nominationData = loadData(url, tables("nominationTable"))
+    val projectDf = programData.join(nominationData, Seq("program_id"), "outer")
```

Comment:
tables should come from application conf. Only the host/ip specific configurations can come from job config